kroma-validator/challenger.go

```diff
@@ -601,8 +601,8 @@ func (c *Challenger) handleChallenge(outputIndex *big.Int, asserter common.Addre
 			// if challenger
 			if isChallenger && c.cfg.ChallengerEnabled {
 				if isOutputDeleted {
```

Comment:
L603-L619 should be like below. The challenger should continue the loop until the challenge status becomes challenger timeout.
```go
				// if output has been already deleted, cancel challenge to refund pending bond
				if isOutputDeleted && status != chal.StatusChallengerTimeout {
					tx, err := c.CancelChallenge(c.ctx, outputIndex)
					if err != nil {
						c.log.Error("failed to create cancel challenge tx", "err", err, "outputIndex", outputIndex)
						continue
					}
					if err := c.submitChallengeTx(tx); err != nil {
						c.log.Error("failed to submit cancel challenge tx", "err", err, "outputIndex", outputIndex)
						continue
					}
				}
```